daemon/policy.go

```diff
@@ -178,19 +220,23 @@ func (h *getPolicyResolve) Handle(params GetPolicyResolveParams) middleware.Resp
 	return NewGetPolicyResolveOK().WithPayload(&result)
 }
 
-func (d *Daemon) enablePolicyEnforcement() {
+func (d *Daemon) updateEndpointPolicyEnforcement(ep *endpoint.Endpoint, enable bool) {
 	d.conf.Opts.Set(endpoint.OptionPolicy, true)
```

Comment:
I'll move the logs to within the block later in this method that checks if options have changed.